README.md

```diff
@@ -37,12 +37,59 @@ This package exposes commands for [laminas-cli](https://docs.laminas.dev/laminas
+- **mezzio:routes:list**: List the application's routing table.
+
+### Routes
+
+#### mezzio:routes:list
+
+This command lists the application's routing table.
+For each route, it prints its name, path, middleware, and any additional options, in a tabular format to the terminal.
+The routes are listed in no particular order, by default.
+
+The command supports several options, listed in the table below.
+
+| Command Long        | Command Short | Description                                                                                                                                                                                                                                                                                                                      |
+|---------------------|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--format`          | `-f`          | Set the format of the command's output. The supported values are `table`, which is the default, and `json`. If you set the format to json, then we recommend using [jq](https://stedolan.github.io/jq/manual/) to query/filter the command's output.                                                                             |
+| `--sort`            | `-s`          | Sort the command's output. The supported values are `name` and `path`.                                                                                                                                                                                                                                                           |
+| `--supports-method` | `-m`          | Accepts a comma-separated list of one or more HTTP methods, and filters out routes which don't support those methods.                                                                                                                                                                                                            |
+| `--has-path`        | `-p`          | Accepts a comma-separated list of one or more paths, and filters out routes with paths that don't match. The paths can be a regular expression, supported by the preg_* functions. For example, "/,/api/ping,*/ping".                                                                                                            |
+| `--has-name`        | `-n`          | Accepts a comma-separated list of one or more names, and filters out routes with names that don't match. The names can be fixed strings, or regular expressions supported by the preg_* functions. For example, "user,user.register,*.register,user*".                                                                           |
```

Comment:
```suggestion
| `--has-name`        | `-n`          | Accepts a comma-separated list of one or more names, and filters out routes with names that don't match. The names can be fixed strings, or regular expressions supported by the "preg_*" functions. For example, "user,user.register,*.register,user*".                                                                           |
```

Fixes `Error: README.md:63:207 MD037/no-space-in-emphasis Spaces inside emphasis markers [Context: "* f”]`